config/config/sc-config.yaml

```
@@ -265,6 +265,8 @@ prometheus:
           memory: 100Mi
   ## the percentage of use the resources are set to hit in 3 days
   predictLinearLimit: 66
+  s3BucketSizeLimit: 80
+  s3BucketSize: 1000
```

```suggestion
  s3BucketPercentLimit: 80
  s3BucketMaxSizeLimit: 1000
```

---

Maybe even this, to show what scale the size limit has.
```suggestion
 s3BucketPercentLimit: 80
  s3BucketGBLimit: 1000
```

---

Also, I think you can put this into the common config instead of duplicating it into both sc and wc config.